### PR TITLE
feat(curriculum): create semantic html quiz

### LINKED ISSUE #56175: [HACKTOBERFEST 2024] - Semantic HTML Quiz
> [!IMPORTANT]
> This issue is part of our 2024 Hacktoberfest initiative. If you are interested in working on this issue, refer to our [top-level tasks](https://github.com/freeCodeCamp/freeCodeCamp/issues/56170) to see if this issue is claimed, or to claim it if it is not.

This issue is to create the `Quiz Name` quiz.

The file you should be working on is found in `/curriculum/challenges/english/25-front-end-development/quiz-name`.

**All quiz questions need to relate to one of these topics:**

- Importance of semantic HTML
- Importance of having good structural hierarchy 
- Difference between presentational and semantic HTML
- emphasis element 
- Idiomatic Text element
- strong element 
- Bring Attention To element
- description lists 
- Inline Quotation element
- Block Quotation element
- Abbreviation element
- Address element
- `<time>`: The (Date) Time element
- The Superscript element
- The Subscript element
- The code element
- `<u>`: The Unarticulated Annotation (Underline) element
- The ruby element
- `<s>`: The Strikethrough element

> [!CAUTION]
> Questions which do not fall under these topics will be rejected.

You are welcome to include code blocks in your questions, but please keep them short (four lines or less).

Please do NOT open a pull request until you have completed all questions. Your quiz should have **exactly 20 questions**.

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #56175

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/quiz-semantic-html/66ed903cf45ce3ece4053ebe.md
Comment (R688t) [inline curriculum/challenges/english/25-front-end-development/quiz-semantic-html/66ed903cf45ce3ece4053ebe.md:21]: HTML5, HyperText Markup Language, is code created that is HTML5 and has stylistic value.

**Review by vignesh1507 — COMMENTED** [review]
Looks fine according to me.

**Comment by a2937** [comment]
@vignesh1507  @R688t  

Please don't review these. All it does is create more noise when PRs are reviewed by non-maintainers.

**Review by a2937 — APPROVED** [review]
That being said, I did learn some new things and enjoyed looking over this quiz and testing it.

**Review by huyenltnguyen — APPROVED** [review]
I'm not sure if you need someone on the curriculum team to review the PR, too, so I'll let you handle the merge 🙂

**Pull request merged**
